openid-4-verifiable-presentations-1_0.md

```diff
@@ -1492,7 +1492,139 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
+
+## Signed and Unsigned Requests
+
+Any OpenID4VP request compliant to this section of this specification can be sent over the W3C Digital Credentials API. Depending on the mechanism used to identify and authenticate the Verifier, the request can be signed or unsigned. This section defines signed and unsigned OpenID4VP requests over the W3C Digital Credentials API.
+
+### Unsigned Request {#unsigned_request}
+
+The Verifier MAY send all the OpenID4VP request data as JSON elements in the `request` API parameter and receives the result in the API's `result` parameter. In this case, the Wallet will use the Verifier origin as asserted by the Browser as the Verifer's Client Identifier.
+
+The unsigned Request Object MUST NOT contain `client_id` and `client_id_scheme` parameters.
+
+### Signed Request {#signed_request}
+
+The Verifier MAY send a signed request, for example, when identification and authentication of the Verifier is required.
+
+The signed Request Object MAY contain all the parameters listed in (#browser_api_request), except `request`. The signed Request Object MUST contain an `expected_origins` parameter. 
+
+Below is a non-normative example of such a request:
+
+```js
+const credential = await navigator.identity.get({
+  digital: {
+    providers: [{
+      protocol: "urn:openid.net:oid4vp",
+      request: {
+        request: "eyJhbGciOiJF..."
+     }
+    }]
+  }
+});
+
+This is an example signed request payload:
+
+<{{examples/digital_credentials_api/signed_request_payload.json}}
+
+The signed request allows the Wallet to authenticate the Verifier using a trust framework other than the Web PKI utilized by the browser. An example of such a trust framework is the Verifier (RP) management infrastructure set up in the context of the eIDAS regulation in the European Union, in which case, the Wallet can no longer rely only on the web origin of the Verifier. This web origin MAY still be used to further strengthen the security of the flow. The external trust framework could, for example, map the Client Identifier to registered web origins.
+
+## Response
+
+Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The Authorization Response is a JavaScript object, where the response parameters as defined for the Response Type are encoded as top-level members in this JavaScript object. 
+
+The following is a non-normative example of an OpenID4VP response that could be received from the W3C Digital Credentials API:
+	
+```js
+{
+  vp_token: "...",
+  presentation_submission: {...}
+}
+```
+
+Note: All mechanisms for cryptographically protecting the OpenID4VP response MAY be utilized with the W3C Digital Credentials API, too.
```

Comment:
I'm not sure what this means... "All mechanisms" seems scary. Can we point to a restricted set of mechanisms?

Reply:
```suggestion
```

we probably can remove this because we already have above
> The value of `response_mode` parameter MUST be `w3c_dc_api` when the response is neither signed nor encrypted and `w3c_dc_api.jwt` **when the response is signed and/or encrypted as defined in (#jarm)**.

and I think this sentence was meant to say what I put in bold above

Reply:
Discussed on today's WG call; people are happy with Kristina's suggestion to remove the sentence.